packages/thea/src/config/substrate/config/polkadot.ts

```diff
@@ -0,0 +1,29 @@
+import { AssetConfig, ChainConfig } from "@moonbeam-network/xcm-config";
+import { BalanceBuilder } from "@moonbeam-network/xcm-builder";
+
+import { ExtrinsicBuilderV2 } from "../builders";
+import { polkadex, polkadot } from "../chains";
+import { dot } from "../assets";
+
+const toPolkadex: AssetConfig[] = [
```

Comment:
polkadot to assethub transfer of dot should be possible

Reply:
We will be addding it later. We are currently adding existing  THEA chains only. I have created an separate issue for it. 
https://github.com/Polkadex-Substrate/polkadex-ts/issues/220